python2.7/Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT: all
```

Comment:
build.sh also need to vary right based on the Dockerfile location.. So i will keep build.sh as it is. and add new build.sh in makefile.. thats fine right?

Reply:
Only one `build.sh` is needed. It already has a parameter for `PYTHON_VERSION`. So if the `PYTHON_VERSION` is 2.7, use `-f python27.Dockerfile`. It is better to have the `Dockerfile` at the root instead of in a subbdirectory. A subdirectory will change the docker build context.